src/main/java/org/broadinstitute/hellbender/tools/exome/germlinehmm/CopyNumberTriState.java

```diff
@@ -40,8 +55,7 @@
     /**
      * Unmodifiable list of alternative alleles order by their corresponding state ordinal.
      */
-    public static List<Allele> ALTERNATIVE_ALLELES =
-            ALL_ALLELES.subList(1, ALL_ALLELES.size());
+    public static List<Allele> ALTERNATIVE_ALLELES = ALL_ALLELES.subList(1, ALL_ALLELES.size());
```

Comment:
I agree, this is not my code, but: ALL_ALLES is defined earlier as an unmodifiable list where REF_ALLELE is clearly the first element. So I think this is fine as is.